addons/sourcemod/scripting/gokz-momsurffix.sp

```diff
@@ -22,7 +22,7 @@ public Plugin myinfo = {
 #define FLT_EPSILON 1.192092896e-07
 #define MAX_CLIP_PLANES 5
 
-#define ASM_PATCH_LEN 24
```

Comment:
This change does not really matter - I don't think ASM_PATCH_LEN is used anywhere anymore, but it still exists in the momsurffix code, so we keep it around anyways.